apps/api/src/app/activity/dtos/shared.dto.ts

```diff
@@ -8,6 +8,15 @@ export enum WorkflowRunStatusDtoEnum {
   ERROR = 'error',
 }
 
+export class TopicResponseDto {
+  @ApiProperty({ description: 'Internal topic identifier' })
+  @IsString()
+  _topicId: string;
```

Comment:
what is the current practice on the underscore internal ids on the dto level? do we expose them, or do we map them to `id` `_topicId`

Reply:
in the inbox routes, we don't do the underscore, but all the other public routes usually for internal IDs have an underscore, check workflows or topics controllers